src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -44,6 +49,14 @@ def test_u_nk(mbar_filename,
 
     assert u_nk.index.names == names
 
+@pytest.mark.parametrize("mbar_filename",
+                         [mbar_filename
+                          for leg in load_bace_example()['data']['complex'].values()
+                          for mbar_filename in leg])
+def test_u_nk_time_reading(mbar_filename, first_time=22.0, last_time=1020.0):
+    u_nk = extract_dHdl(mbar_filename, T=300)
```

Comment:
Do you really want to test `extract_dHdl()` here and not `extract_u_nk()`?

Reply:
You are right, it was a wild copy/paste :-)
I changed it in the last commit!